### PR TITLE
setup project

### PR DESCRIPTION
**Pull request by zemola**
### Leaderboard: setup project

Setting up the project with the below requirement

- Using git-flow
- Creating an npm project with webpack. 
- Writing plain HTML markup with minimum styling (just to make the layout).

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+            <div class="score-board">
+              <h2>Recent scores</h2>
+              <button type="button" class="btn refresh-btn">Refresh</button>
+            </div>
+            <div class="display-score">
+              <ul class="score-list" id="score">
+                <li>AZEEM ADE : 29</li>
+                <li>AZEEM BAYO : 99</li>
+                <li>AZEEM ADEBAYO: 89</li>
+                <li>AZEEM ADISA: 36</li>
```
Comment (mikethreels) [inline dist/index.html]: - [ ] good job creating the layout with HTML 👍  however according to the requirements HTML should be used just to make the layout, to fill in the list you should use ES6 modules, with import and export.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Azeem, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: dist/index.html (hunk 1 above)
Comment (zemola) [reply dist/index.html]: Noted,
Thank you for your observation.

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @zemola, 

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**